freva-client/src/freva_client/query.py

```diff
@@ -831,22 +893,19 @@ def _get(self, url: str, **kwargs: Any) -> Optional[requests.models.Response]:
             logger.warning(msg)
         return None
 
-    def _put(
+    def _post(
```

Comment:
I think, rather than adding extra methods that do kind of similar things, you could do one method that gets the request type to rule them all. So instead of `reuests.get, requests.put` etc you use `requests.Request("GET",...)` etc.